### DISCUSSION TITLE
High-resolution EBSD

### DISCUSSION THREAD

**Discussion by pablobruno1** [mtex-toolbox/mtex#1384] [Ask Anything]
Hello,
Is there in MTEX the possibility to assess high-resolution EBSD strain?

**Comment by Tijmenvermeij** [comment]
I guess it depends on what you mean by "assess". Mtex does not have the capability to calculate elastic strains through correlation of EBSD patterns. However, if you already have the strain data, you could use Mtex for further analysis since it has strain and stress tensor objects implemented...
